tools/clang/include/clang/SPIRV/ModuleBuilder.h

```diff
@@ -67,16 +67,35 @@ class ModuleBuilder {
   /// for the basic block. On failure, returns zero.
   uint32_t createBasicBlock(llvm::StringRef name = "");
 
+  /// \brief Adds the basic block with the given label as a successor to the
+  /// current basic block.
+  void addSuccessor(uint32_t successorLabel);
+
+  /// \brief Sets the merge target to the basic block with the given <label-id>.
+  /// Must make sure the current basic block contains an OpSelectionMerge or
```

Comment:
s/Must make sure/The caller must make sure/

The method does not assert this. So, I'm guessing the responsibility lies on the caller.